examples/gno.land/r/gnoland/boards2/v1/boards.gno

```diff
@@ -32,6 +32,7 @@ func init() {
 	gPerms = initRealmPermissions(
 		"g16jpf0puufcpcjkph5nxueec8etpcldz7zwgydq", // @devx
 		"g1manfred47kzduec920z88wfr64ylksmdcedlf5", // @moul
+		"g1mtmrdmqfu0aryqfl4aw65n35haw2wdjkh5p4cp", // @test123 -> for txtar
```

Comment:
we should not do this. But i think Txtar can pretend being @devx or @moul, cc @gfanton.

Reply:
It would be great if we could delete this line, otherwise we should remove the `@test123` user each time after deploying boards